recipes/kart/meta.yaml

```
@@ -0,0 +1,43 @@
+{% set version = "2.5.2" %}
+
+package:
+  name: kart
+  version: {{ version }}
+
+source:
+  url: https://github.com/hsinnan75/Kart/archive/{{ version }}.tar.gz
+  sha256: '99f28beb55ebd118c7e7e27507805d9725f0e490557107cf7058351dbed05981'
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('c') }}
+  host:
+    - zlib
+    - bzip2
+    - xz
+    - boost-cpp
+  run:
+    - zlib
+    - bzip2
+    - xz
+    - boost-cpp
+
+test:
+  commands:
+    - kart
```

Maybe a better test might be `kart -version | grep '{{ version }}'`

---

Sounds good.
```suggestion
kart --version | grep -qF '{{ version }}'
```
with `-F` we can compare the `.`s in the version string correctly.

---

Thanks for the suggestion. However, it produces nothing with the commands. I used "kart -h" instead.

---

Do you mean `kart` or the test command?
For `kart`, I assumed it did because of https://github.com/hsinnan75/Kart/blob/2.5.2/src/main.cpp#L155.
As for the test command, yes, no text output. The output is just exitcode 0 if it succeeded and 1 if not (i.e., tests will fail)
If you want to see the output as well as test it in a single command, you can use
```
kart --version | tee /dev/stderr | grep -qF '{{ version }}'
```
but seeing the output is not needed here.
You can also do
```suggestion
    - kart -h
    - kart --version
    - kart --version | grep -qF '{{ version }}'
```
if you want to test the output and see everything in the build log.

---

Thank you! I got it. I'll use those commands for testing new packages.